template/page-websocket-api-tool.html.jinja

```diff
@@ -100,49 +100,71 @@
         </div>
         <div class="modal-body">
 
+          {% for conn in [
```

Comment:
Changed this from a hardcoded set of HTML elements to a for-loop across a hardcoded JSON-like list because it makes it easier to identify the unique parts of each element and reduces opportunity for typos/inconsistency in the parts of each HTML entry that are the same.

Future work could theoretically move this out to a config file or something. (That's likely how we'd want to do it on Redocly anyway.)